src/pages/admin/programs/components/program-form/ProgramForm.tsx

```
@@ -360,6 +432,56 @@ export const ProgramForm = forwardRef<ProgramFormRef, ProgramFormProps>(
+                                    <PlusIcon className={styles['plus-icon']} />
+                                </Button>
+                            </div>
+                            <div className={styles['sections-divider']} />
+                        </>
+                    )}
+
+                    {hasSections && (
+                        <div className={styles['sections-list']}>
+                            {formState.sections.map((section, index) => (
+                                <React.Fragment key={section.id ?? `${section.template}-${index}`}>
+                                    {true ? (
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)